.gitignore

```diff
@@ -1,2 +1,17 @@
+# Mainflux
 build
-site/
+.vscode
```

Comment:
This is specific for the local dev env. Please remove this. (You can add it to your global `.gitignore` on your machine).